File: cpp/1171.cpp
#include <iostream>
#include <string>
#include <map>

using namespace std;

int main()
{
    map<int, int> numbers;
    int n, number;
    cin >> n;

    for (n; n > 0; n--) {
        cin >> number;

        if (numbers.find(number) == numbers.end()) {
            numbers[number] = 0;
        }

        numbers[number]++;
    }

    for (auto const pair: numbers) {
        cout << pair.first << " aparece " << pair.second << " vez(es)" << '\n';
    }

    return 0;
}

File: py/1171.py
n = int(input())
numbers = []
each = set()

while n > 0:
    number = int(input())
    numbers.append(number)
    each.add(number)
    n -= 1
    
each = sorted(each)

for i in each:
    print(f'{i} aparece {numbers.count(i)} vez(es)')